services/net/reporting/ReportingManager.cs

```diff
@@ -361,7 +361,8 @@ private async Task<(API.Areas.Services.Models.ReportInstance.ReportInstanceModel
         }
 
         // Generate a new instance, or accumulate new content each time the report is run.
-        if (reportInstanceModel == null || !(!report.Settings.Content.ClearOnStartNewReport && !report.Settings.Content.CopyPriorInstance))
+        if (reportInstanceModel == null || reportInstanceModel.SentOn.HasValue
```

Comment:
Fix bug where the report wasn't being generated if the prior was already sent.  I'm hoping this doesn't introduce a regression issue, as I don't understand how it would have worked in other cases.